server/grpc_service.go

```diff
@@ -1621,7 +1621,12 @@ func (s *GrpcServer) ScatterRegion(ctx context.Context, request *pdpb.ScatterReg
 		return nil, err
 	}
 	if op != nil {
-		rc.GetOperatorController().AddOperator(op)
+		if !rc.GetOperatorController().AddOperator(op) {
```

Comment:
How about adding a retry for it and returning an error if it finally fails? /cc @nolouch @bufferflies

Reply:
I think it should depend on a caller.  Caller should retry with back off.

Reply:
I aggre with the mittalrishabh, the producer should response with the error.